config/application.rb

```diff
@@ -16,8 +16,12 @@ class Application < Rails::Application
     config.analytics_tracking_id = ENV['GA_TRACKING_ID']
     config.analytics_test_tracking_id = ENV['GA_TEST_TRACKING_ID']
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**','**','*.{rb,yml}')]
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
+    config.filter_parameters << :password
+    config.filter_parameters << :password_confirmation
+
+    Raven.configure do |config|
+      config.dsn = 'https://b9b9fe3fe0e84d95a2a2b513b7f859a7:e038e55809d64b2ba482e23031d0e21f@sentry.io/236517'
```

Comment:
hardcoded?

Reply:
I guess the worst someone could do is flood us with errors, but you're right, should move this to a config.